pom.xml

```diff
@@ -53,7 +53,9 @@
       <module>artemis-jakarta-server</module>
       <module>artemis-journal</module>
       <module>artemis-ra</module>
+      <module>artemis-ra-rar</module>
```

Comment:
The doc additions speak about needing to build your own rar, and directs to the examples present for doing that. The 2 new modules being added at the root seem to be doing much the same thing. I'm not clear why both exist.

Reply:
see my comment here https://github.com/apache/activemq-artemis/pull/3999#issuecomment-1090719349